xapian-core/tests/api_queryparser.cc

```diff
@@ -1725,6 +1725,84 @@ DEFINE_TESTCASE(qp_range4, !backend) {
     return true;
 }
 
+static const test test_value_filesizerange1_queries[] = {
+    { "10K..20K", "0 * VALUE_RANGE 1 10240 20480" },
+    { "10..50M", "0 * VALUE_RANGE 1 10485760 52428800" },
+    { "10K..20", "Unknown range operation" },
+    { "10A..20A", "Unknown range operation" },
+    { "10..20H", "Unknown range operation" },
```

Comment:
You aren't testing any cases without suffices.

You aren't testing with empty range ends.

And you're only testing integer values.